modules/order/src/Plugin/Field/FieldFormatter/OrderTotalSummary.php

```diff
@@ -76,10 +76,15 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
     $order = $items->getEntity();
-    return [
-      '#theme' => 'commerce_order_total_summary',
-      '#totals' => $this->orderTotalSummary->buildTotals($order),
-    ];
+
+    $element = [];
+    foreach ($items as $delta => $item) {
```

Comment:
Hello! Yes, I see your logic. Mine was: normally it should just 1 item, but if somebody "up there went crazy" and made cardinality of this field more than 1, then it's what I am expected to do: print total X times. However, it's such an edge case that i have no preference between my logic and yours.